src/dmd/chkformat.d

```diff
@@ -145,6 +145,7 @@ bool checkPrintfFormat(ref const Loc loc, scope const char[] format, scope Expre
         auto tnext = t.nextOf();
         const c_longsize = target.c.longsize;
         const isLP64 = global.params.isLP64;
```

Comment:
`dmd/chkformat.d(147:15)[warn]: Variable isLP64 is never used`

Reply:
Yeah, with this and the other PR combined, all uses of isLP64 are removed.

Reply:
So I guess merge #12365 first, and I'll rebase this ontop with dead code eliminated.

Reply:
merged that one.

Reply:
Rebased this one.